src/style.css

```
@@ -0,0 +1,84 @@
+html {
+  --form-shadow: 0 1px 8px 0 #3d4e611a;
+  --form-shadow-hover: 0 3px 8px 0 #3d4e6133;
+  --color-form: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  font-family: "Avenir", serif;
+  font-weight: 300;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  margin: 20px 8px;
+}
+
+.searchbar__input {
+  width: 100%;
+  box-sizing: border-box;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  box-shadow: var(--form-shadow);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: var(--color-form);
+  font-family: inherit;
+}
+
+.searchbar__input--big {
+  margin-bottom: 20px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  background-size: 19px 19px;
+  background-position-x: 26px;
+  background-position-y: 25px;
+  padding-left: 62px;
+}
+
+.searchbar__input--small {
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  background-size: 11px 11px;
+  background-position-x: 13px;
+  background-position-y: 15px;
+  padding-left: 33px;
+}
+
+.searchbar__input--big:hover,
+.searchbar__input--big:focus,
+.searchbar__input--small:hover,
+.searchbar__input--small:focus {
```

```suggestion
.searchbar__input:hover,
.searchbar__input:focus {
```
no need to list 4 selectors - you can apply the same with 2

but the text color shouldn't change on hover, and on focus, you should add a gradient
so remove here `.searchbar__input:focus` too here